docs/en/solutions/How_to_Migrate_MySQL_57_to_80.md

```
@@ -0,0 +1,2409 @@
+DATABASES="ALL"                   # "ALL" to auto-detect
+```
+
+**Usage:**
+```bash
+vi 00-pre-migration-check.sh       # Edit configuration
+chmod +x 00-pre-migration-check.sh
+./00-pre-migration-check.sh
+```
+
+**Expected Output:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)